agents.md

```
@@ -1,81 +1,125 @@
+```bash
+export NVM_DIR="${NVM_DIR:-$HOME/.nvm}" && [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" && nvm use && pnpm -v
+```
+
+Replace `pnpm -v` with the real command (e.g. `pnpm run validate:all`). If `NVM_DIR` is non-standard on the machine, set it before sourcing.
+
+- Do not run `npm` commands — this repo uses pnpm workspaces.
+- Install deps: `pnpm install`
+- Wipe turbo cache: `pnpm run turbo:cache:wipe`
+- Generate diagrams (monorepo, game-client, backoffice-client): `pnpm run diagrams:all`
+
+### All packages scripts
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)